frontend/src/components/Invitations.js

```diff
@@ -11,21 +11,9 @@ class Invitations extends Component {
     };
   }
 
-  handleFirstNameChange = (e) => {
+  handleInputChange = (e) => {
```

Comment:
Love the destructuring here.  You may want to comment this line of code incase one of your teammates doesn't understand this or if someone find the repo and reads over.